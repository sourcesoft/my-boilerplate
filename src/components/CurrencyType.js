import styled from 'styled-components';

export const CurrencyType = styled.div`
  color: #1f1e1f;
  font-size: 1.5rem;
  border: 1px solid #000;
  padding: 15px 25px;
  width: 100%;
  margin-bottom: 30px;
`;

export default CurrencyType;
