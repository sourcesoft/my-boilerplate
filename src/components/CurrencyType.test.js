import React from 'react';
import CurrencyType from './CurrencyType.js';
import { shallow } from 'enzyme';

test('should render CurrencyType with children', () => {
  const comp = shallow(<CurrencyType>Test</CurrencyType>).html();
  expect(comp).toMatchSnapshot();
});
