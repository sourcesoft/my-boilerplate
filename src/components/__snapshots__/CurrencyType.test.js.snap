// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render CurrencyType with children 1`] = `"<div class=\\"sc-bdVaJa gEOrok\\">Test</div>"`;
